perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -261,6 +270,14 @@ sub open311_pre_send {
         my $text = $row->detail . "\n\nPrivate comments: $private_comments";
         $row->detail($text);
     }
+
+    if (my $comment_id = $row->get_extra_metadata('echo_report_reopened_with_comment')) {
+        my $comment = FixMyStreet::DB->resultset('Comment')->find($comment_id);
+        if ($comment && $comment->text) {
+            my $text = "Closed report has a new comment: " . $comment->text . "\n\n" . $row->detail;
+            $row->detail($text);
```

Comment:
I'm not sure if the new event will easily have access to the original event text so this should probably be an addition to the detail, rather than replacing it?

Reply:
Dur, it does, at the end!